incubator/ark-server/templates/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  namespace: {{ .Values.namespace.name }}
+  name: ark
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: ark
+    spec:
+      restartPolicy: Always
+      serviceAccountName: {{ template "ark-server.sa" . }}
+      containers:
+        - name: ark
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          command:
+            - /ark
+          args:
+            - server
```

Comment:
You'll want to add
```
- --namespace
- {{ <a variable for the namespace in which backups, schedules, restores, and the Ark config go> }}
```

So you'll probably need 1 variable for the namespace where the server lives, and a second variable for the namespace where the backups etc go.

Reply:
`- {{ <a variable for the namespace in which backups, schedules, restores, and the Ark config go> }}`

You wrote also  `ark config`, but I read the new yaml files and it remeains in heptio-ark-server namespace, right?

Reply:
No, the config remains in `heptio-ark` along with backups, schedules, and restores.